source/documentation/tax-calculations.html.md

```diff
@@ -25,6 +25,8 @@ When a customer requests to view an existing calculation, the software should ca
 
 If the software retrieves a calculation, it will cover only the period up to their last submission date, not the calculation retrieval date. For example, if a customer submitted their update to 1 July 2023 and they requested a calculation on 30 July 2023, they will receive a calculation only up to 1 July 2023.
 
+Before presenting your customer with an in-year tax calculation, the [minimum functionality standards](https://developer.service.hmrc.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/documentation/how-to-integrate.html#minimum-functionality-standards) state that a disclaimer must be presented. This disclaimer is returned through the [Individual Calculations API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individual-calculations-api/6.0) as ‘This calculation is only based on information HMRC have received about your income and expenses to 20XX-XX-XX. This may change as we receive further information about you during the tax year’. Software providers can choose to use this exact disclaimer or design their own.
```

Comment:
A few suggestions / questions:

- Can `https://developer.service.hmrc.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/documentation/how-to-integrate.html#minimum-functionality-standards` be updated to `/guides/income-tax-mtd-end-to-end-service-guide/documentation/how-to-integrate.html#minimum-functionality-standards` like in [here](https://github.com/search?q=repo%3Ahmrc%2Fincome-tax-mtd-end-to-end-service-guide%20%2Fguides%2Fincome-tax-mtd-end-to-end-service-guide%2Fdocumentation%2Fhow-to-integrate.html%23minimum-functionality-standards&type=code)
- Can `https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individual-calculations-api/6.0` be updated to `/api-documentation/docs/api/service/individual-calculations-api` i.e. remove `https://developer.service.hmrc.gov.uk` and `/6.0` so we don't have to worry about broken links when these versions are retired. It will default to the latest version
- `This calculation is only based on information HMRC have received about your income and expenses to 20XX-XX-XX. This may change as we receive further information about you during the tax year` is in single quotation marks but the recommended content is in double. Does that matter?
- In this sentence, `Software providers can choose to use this exact disclaimer or design their own.`, the word `providers` is not present in the recommended content. Should this be removed?

Reply:
Thanks Taiwo, I will update the URLs. 
- I used single quotes to remain consistent with the messages that are used further up the page.
- I added the word 'providers' for clarity - Software itself cannot choose to do anything :)

Reply:
I should add that the rest of the links on this page all use direct links rather than relative paths, so maybe we should stay 
consistent?